Makefile

```diff
@@ -1,5 +1,5 @@
-PYTHON_VERSION_VENV ?= python3.9
-TOX_ENVLIST ?= python3.9
+PYTHON_VERSION_VENV ?= python3.11
+TOX_ENVLIST ?= python3.11
```

Comment:
I would keep 3.9 as the default, otherwise you won't notice that you're using incompatible code until you run all tests (I usually only run the test-suite for one python and let CI handle the rest)